### PR TITLE
✨️Implement methot to retrieve conditional availabilities by ids

### PR DESCRIPTION
**Pull request by gengjozsef**
- Implement Facade methods
- Implement Repository methods
- Implement Unit Test
- Code cleanup

### CODE REVIEW

File: bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacade.php [hunk 1]
```diff
@@ -158,4 +158,18 @@ public function persistConditionalAvailability(
     ): ConditionalAvailabilityResponseTransfer {
         return $this->getFactory()->createConditionalAvailabilityWriter()->persist($conditionalAvailabilityTransfer);
     }
+
+    /**
+     * Specifications:
+     * - Retrieves conditional availabilities by ids
+     *
+     * @param array<int>$conditionalAvailabilityIds
```
Comment (daniel-rose) [inline bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacade.php:166]:
```suggestion
     * @param array<int> $conditionalAvailabilityIds
```

File: bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacadeInterface.php [hunk 2]
```diff
@@ -143,4 +143,16 @@ public function findConditionalAvailabilities(
      * @return array<int>
      */
     public function getConditionalAvailabilityIdsByProductConcreteIds(array $productConcreteIds): array;
+
+    /**
+     * Specifications:
+     * - Retrieves conditional availabilities by ids
+     *
+     * @param array<int>$conditionalAvailabilityIds
```
Comment (daniel-rose) [inline bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacadeInterface.php:151]:
```suggestion
     * @param array<int> $conditionalAvailabilityIds
```

File: bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Persistence/ConditionalAvailabilityRepositoryInterface.php [hunk 3]
```diff
@@ -52,4 +52,13 @@ public function findConditionalAvailabilityPeriodsByFkConditionalAvailability(
      * @return array<int>
      */
     public function getConditionalAvailabilityIdsByProductConcreteIds(array $productConcreteIds): array;
+
+    /**
+     * @param array<int>$conditionalAvailabilityIds
```
Comment (daniel-rose) [inline bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Persistence/ConditionalAvailabilityRepositoryInterface.php:57]:
```suggestion
     * @param array<int> $conditionalAvailabilityIds
```

**Review by daniel-rose — CHANGES_REQUESTED** [review]

**Review by daniel-rose — APPROVED** [review]

**Review by julianzimmermann — APPROVED** [review]

File: bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacade.php (hunk 1 above)
Comment (gengjozsef) [reply bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacade.php]: Thank you for the close attention @daniel-rose i implemented your suggestions

File: bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacadeInterface.php (hunk 2 above)
Comment (gengjozsef) [reply bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Business/ConditionalAvailabilityFacadeInterface.php]: Thank you for the close attention @daniel-rose i implemented your suggestions

File: bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Persistence/ConditionalAvailabilityRepositoryInterface.php (hunk 3 above)
Comment (gengjozsef) [reply bundles/conditional-availability/src/FondOfImpala/Zed/ConditionalAvailability/Persistence/ConditionalAvailabilityRepositoryInterface.php]: Thank you for the close attention @daniel-rose i implemented your suggestions

**Pull request merged**